imap_processing/cdf/config/imap_idex_l1_variable_attrs.yaml

```diff
@@ -0,0 +1,420 @@
+  SCALETYP: linear
+  MONOTON: INCREASE
+  TIME_BASE: J2000
+  TIME_SCALE: Terrestrial Time
+  REFERENCE_POSITION: Rotating Earth Geoid
+
+sample_rate_base: &sample_rate_base # FloatAttrs
+  DISPLAY_TYPE: no_plot
+  CATDESC: ""
+  FIELDNAM: ""
+  FILLVAL: -1.0e+31 # DOUBLE FILL VAL
+  VALIDMIN: *sample_rate_min
+  VALIDMAX: *sample_rate_max
+  DEPEND_0: epoch
+  FORMAT: F64.5
+  LABLAXIS: Time
+  UNITS: microseconds
+  VAR_TYPE: support_data # Not in attrs file
+  VAR_NOTES:  The number of microseconds since the event.
+              0 is the start of data collection, negative
+              numbers represent data collected prior to a dust event
+
+trigger_base: &trigger_base # ScienceAttrs
+  VALIDMIN: 0
+  VALIDMAX: *int_maxval
+  DISPLAY_0: epoch
+  DISPLAY_TYPE: no_plot
+  CATDESC: ""
+  FIELDNAM: ""
+  VAR_TYPE: support_data
+  FILLVAL: *int_fillval
+  LABLAXIS: ""
+  FORMAT: I12
+  UNITS: ""
+
+# TODO: All of these have var_notes in idex_cef_attrs. Should I add these?
+low_sr_attrs:
+  <<:  *sample_rate_base
+  CATDESC: Low sample rate time steps for a dust event.
+  FIELDNAM: Low Sample Rate Time
+  FILLVAL: *int_fillval
+
+high_sr_attrs:
+  <<:  *sample_rate_base
+  CATDESC: High sample rate time steps for a dust event.
+  FIELDNAM: High Sample Rate Time
+
+tof_high_attrs:
+  <<: *l1_tof_base
+  CATDESC: Time of flight waveform on the high-gain channel
+  FIELDNAM: High Gain Time of Flight
+  LABLAXIS: TOF High Ampl.
+
+tof_mid_attrs:
+  <<: *l1_tof_base
+  CATDESC: Time of flight waveform on the mid-gain channel
+  FIELDNAM: Mid Gain Time of Flight
+  LABLAXIS: TOF Mid Ampl.
+
+tof_low_attrs:
+  <<: *l1_tof_base
+  FILLVAL: *int_fillval
+  CATDESC: Time of flight waveform on the low-gain channel
+  FIELDNAM: Low Gain Time of Flight
+  LABLAXIS: TOF Low Ampl.
+
+
+target_low_attrs:
+  <<: *l1_target_base
+  CATDESC: Target low charge sensitive amplifier waveform
+  FIELDNAM: Low Target Signal
+  LABLAXIS: Low Target Ampl.
+
+target_high_attrs:
+  <<: *l1_target_base
+  CATDESC: Ion grid charge sensitive amplifier waveform
+  FIELDNAM: High Target Signal
+  LABLAXIS: High Target Ampl.
+
+ion_grid_attrs:
+  <<: *l1_target_base
+  CATDESC: Ion grid charge sensitive amplifier waveform data
+  FIELDNAM: Ion Grid Signal
+  LABLAXIS: Ion Grid Ampl.
+
+# Trigger Stuff
+
+event_number:
+  <<: *trigger_base
+  CATDESC: The unique number assigned to the impact by the FPGA
+  FIELDNAM: Event Number
+  VALIDMAX: 65535
+  LABLAXIS: Event Number
+  UNITS: ""
+
+tof_high_trigger_level:
+  <<: *trigger_base
+  VALIDMAX: 1023
+  FIELDNAM: TOF High Trigger Level
+  CATDESC: Trigger level for the TOF High Channel
+  LABLAXIS: Level
+  UNITS: ""
+
+tof_high_trigger_num_max_1_2:
+  <<: *trigger_base
+  VALIDMAX: 2047
+  FIELDNAM: TOF High Double Pulse Max Samples
+  CATDESC: Maximum number of samples between pulse 1 and 2 for TOF. High double pulse triggering.
+  LABLAXIS: Samples Number
+  UNITS: "" # "samples" in idex_packet_parser...?
```

Comment:
I think `samples` is fine in these? Perhaps `counts` would also work? This really is a question for the IDEX instrument team in what they want to see, I think. @bryan-harter do you have an idea?